src/modules/Store.js

```diff
@@ -0,0 +1,34 @@
+export default class Store {
+  constructor() {
+    this.data = [];
+    this.scoresUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/266Uox8RtUVX6YlFdld7/scores/';
+  }
+
+  addScore(user, score) {
+    this.obj = {
+      user,
+      score,
+    };
+
+    const promise = fetch(this.scoresUrl, {
+      method: 'POST',
+      body: JSON.stringify(this.obj),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json().result);
+    return promise;
+  }
+
+    #privateSetData() {
+    this.data = fetch(this.scoresUrl)
+      .then((response) => response.json())
+      .then((json) => (json.result));
+  }
+
+    getData() {
+      this.#privateSetData();
+      return this.data;
+    }
```

Comment:
- [ ] According to the project requirement, all API calls should be done with `async await` method, so change all your `.then` methods to `async await`  :+1: